test/action_controller/json/include_test.rb

```diff
@@ -4,6 +4,10 @@ module ActionController
   module Serialization
     class Json
       class IncludeTest < ActionController::TestCase
+        INCLUDE_STRING = 'posts.comments'
+        INCLUDE_HASH = { posts: :comments }
+        DEEP_INCLUDE = 'posts.comments.author'
```

Comment:
A followup PR with a deep include hash would be good, but no need to do more work in this pr